src/Sylius/Bundle/ShopBundle/templates/shared/form/address/province.html.twig

```diff
@@ -1,13 +1,10 @@
 {% set form = hookable_metadata.context.form %}
 
-<div data-url="{{ path('sylius_shop_ajax_render_province_form') }}">
+<div>
     {% if form.provinceCode is defined %}
         {{ form_row(form.provinceCode, sylius_test_form_attribute('province-code')|sylius_merge_recursive( {'attr': {'class': ''}})) }}
+        {{ form_errors(form.provinceCode) }}
```

Comment:
Isn't this included in the `form_row` above?

Reply:
You are right